### DISCUSSION TITLE
Record motion/events AND continous record

### DISCUSSION THREAD

**Discussion by SoftwareSchlosser** [blakeblackshear/frigate#7270] [Ask A Question] [answered]
Is it possible to record events upon detected motion/objects and have a continous recording of e.g. the last 24 hours?

TL;DR
To prevent too many false positives, my motion detection doesn't get triggered by very short movements or small objects. That way some interesting footage might be missing of course and it would be good to be able to review the footage of e.g. the last 24 manually especially if you know something that happened is missing in the detected events.

Adding a camera twice as a workaround is certainly possible but I wonder if there's a better way of doing that?

**Comment by NickM-27** [comment]
It is possible to have recordings for events (object detections) and keep those for x amount of days

And it's possible to record 24/7 keeping all segments, motion segments, or segments with objects and keep those for y amount of days

**Reply by SoftwareSchlosser** [reply]
Sorry, I think my explanation was't quite good ;)

This is my record config:
```
record:
  enabled: True
  retain:
    days: 14
    mode: motion
  events:
    retain:
    default: 90
    mode: active_objects
```

So I keep 90 days of footage for detected objects and 14 days for any motion, no matter if any objects have been detected.
Now is there a way to additionally have continous recordings (mode: all) and retain those for e.g. only a few hours/days?

**Reply by NickM-27** [reply, marked as answer]
No, there is not support for multiple 24/7 recording modes. There is a pinned feature request already for this.

**Reply by SoftwareSchlosser** [reply]
Thank you for the quick response! 👍
